tests/pages/navigation-modal.js

```diff
@@ -1,24 +1,11 @@
 import {
-  page,
+  interactor,
   isVisible,
-  clickable,
-  computed
-} from '@bigtest/interaction';
+  clickable
+} from '@bigtest/interactor';
 
-// the current implementation of `isPresent` throws if the root does not exist
-const isRootPresent = () => {
-  return computed(function () {
-    try {
-      return !!this.$root;
-    } catch (e) {
-      return false;
-    }
-  });
-};
-
-@page class NavigationModal {
-  exists = isRootPresent();
-  isVisible = isVisible();
+@interactor class NavigationModal {
+  isModalVisible = isVisible();
```

Comment:
`isVisible` is one of the new default properties on Interactors. We shouldn't need to do this part anymore

Reply:
What does that mean?

Reply:
Interactors have some default properties now. `isPresent`, `isVisible`, `text`, `value`, etc. Basically the property helpers that can be called with no arguments don't have to be.

Reply:
Fixed this one 👍